pkg/cluster/manager.go

```diff
@@ -0,0 +1,1886 @@
+package cluster
```

Comment:
The license header is missing.

Reply:
https://github.com/pingcap/tiup/pull/587/commits/5d1ad886de4c930d5cd279b770e3bf3796403ff7